### DISCUSSION TITLE
Print string without quotes

### DISCUSSION THREAD

**Discussion by std-any-emplace** [nlohmann/json#2721] [Q&A] [answered]
Is there a way to print a string without any surrounding quotes when `dump()`ing an object?

Example:
```
{
  "a": 1,
  "b": "not like this",
  "c": like this
}
```

**Comment by nlohmann** [comment, marked as answer]
No, this is not possible as dump is meant to produce valid JSON.

**Reply by std-any-emplace** [reply]
I know that it's not valid JSON, but it is still a pity. I was trying to construct an object containing a function name and pass that object to some javascript code to be executed (QWebEnginePage::runJavaScript).
